backend/database/wrapper/stat_math.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import func
+
+from backend.database.objects import PlayerGame
+
+def safe_divide(sql_value):
+    return func.greatest(sql_value, 1)
+
+
+def get_boost_efficiency():
+    wasted_collection = PlayerGame.wasted_collection - PlayerGame.num_stolen_boosts * 50
```

Comment:
This num_stolen_boosts * 50 makes the definition different than:
> Wasted collected boost + wasted boost at speed) / (boost used)
no?

Reply:
There were complaints from high level players that "wasted boost" is misleading because they steal boost a lot. Which causes wasted boost to increase, even though that's not a bad thing to do. So this is a naive attempt at fixing it I guess.

Reply:
I see 2 separate boost values:
- *_Usage_ efficiency:* `wasted_boost_usage / total_usage`
  - More relevant to me, baggage free and benefits from normalization
- *_Collection_ efficiency:*` `boost_lost_to_overflow / total_collected`
 - Less relevant to me, baggage, doesn't benefit from normalization as much

My 2 cents :)